src/Application/Components/NewsRoom/NewsRoom.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const NewsRoom = React.createClass({
+  propTypes: {
+    articles: React.PropTypes.array.isRequired,
+  },
+
+  render() {
+    let count = 0;
+    return (
+      <div id="news-zone">
+        <h3>News Room</h3>
+        <div id="news-queue">
+          <ul>
+            {this.props.articles.map(function (article) {
+              count++;
+              return <li key={count}>{article}</li>;
+            })}
```

Comment:
no research required, just put everything within `<div id="news-queue">...</div>` as the return value of some `renderNewsQueue` method, and here have 
```
<div id="news-zone">
  <h3>News Room</h3>
  {this.renderNewsQueue()}
</div>
```